src/components/batch-selection/batch-selection.test.js

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import { mount } from "@cypress/react";
+import BatchSelection from "./batch-selection.component";
+import IconButton from "../icon-button/icon-button.component";
+import { positionOfElement } from "../../../cypress/support/helper";
+import Icon from "../icon/icon.component";
+
+import {
+  batchSelectionCounter,
+  batchSelectionComponent,
+  batchSelectionButtonsByPosition,
+} from "../../../cypress/locators/batch-selection/index";
+
+context("Tests for BatchSelection component", () => {
+  describe("check BatchSelection component count", () => {
+    it.each(["0", "10", "100"])(
+      "check BatchSelection component %s selected Count",
+      (selectedCount) => {
+        mount(<BatchSelection selectedCount={selectedCount} />);
+        batchSelectionCounter().should(
+          "have.text",
+          `${selectedCount} selected`
+        );
+      }
+    );
+
+    it("should check hidden batch selection", () => {
+      mount(<BatchSelection hidden />);
+      batchSelectionComponent().should("have.attr", "hidden");
+      batchSelectionComponent().should("not.be.visible");
+    });
+
+    it("should check disabled batch selection", () => {
+      mount(<BatchSelection disabled />);
+      batchSelectionComponent().should("have.attr", "disabled");
+    });
+
+    it.each([
+      ["dark", "rgb(0, 51, 73)"],
+      ["light", "rgb(179, 194, 200)"],
+      ["white", "rgba(0, 0, 0, 0)"],
+    ])(
+      "check BatchSelection component %s colorTheme and it uses %s as a background color",
+      (colorTheme, backgroundColor) => {
+        mount(<BatchSelection colorTheme={colorTheme} />);
+        batchSelectionComponent().should(
+          "have.css",
+          "background-color",
+          backgroundColor
+        );
+      }
+    );
+  });
+
+  describe("check BatchSelection button position", () => {
```

Comment:
this test is about being able to focus those buttons inside the `BatchSelection`

Reply:
the same question :)